lektor/admin/modules/api.py

```diff
@@ -2,16 +2,22 @@
 import posixpath
 
 import click
-from flask import Blueprint, jsonify, request, g, current_app
-
-from lektor._compat import iteritems, itervalues
-from lektor.utils import is_valid_id
+from flask import Blueprint
```

Comment:
Which tool is causing these import splits? I don't like this idea.

Reply:
That's `reorder_python_imports`. What don't you like?

Reply:
It's done so diffs will be much simpler, you can see an added/removed line rather than hunt for a word in a long list.

Reply:
I don't like the noise it produces:

```python
# reorder_python_imports
from flask import abort
from flask import Blueprint
from flask import g
from flask import redirect
from flask import render_template
from flask import request
from flask import url_for

# black (don't know whether or not it sorts the functions alphabetically)
from flask import (
    Blueprint,
    render_template,
    abort,
    redirect, request,
    g,
    url_for,
)
# Don't know whether black sorts or not
```

I think the vanilla black style is much better (probably because it's similar to what I'm used to and that it's less verbose). I guess it also has the benefit of producing a smaller diff if we ever rename a file.

Reply:
Hmm okay, I'll revert the two commits, remove `reorder_python_imports` and redo them, though then we won't get the automatic separation of third party packages, consistent ordering, etc.

Reply:
In other (large, unrelated) repos I've worked with, reorder-python-imports also has the effect of automatically fixing some flake8 errors, such as deduping imports, and ensuring the correct import order, which can introduce bugs, even if that's rare. Admittedly, that's less of a problem here since Lektor is small in comparison, and in better shape.

I also didn't like splitting imports like this (or black, for that matter), but they've both grown on me a lot. My vote would be to keep reorder-python-imports. This single diff is a little noisy (thus the .git-blame-ignore-revs), but I think it's worth it. I now don't blink at the import lines that look like this. My 2 cents.

Reply:
I think I typed my comment as you were typing your last one @skorokithakis . For the record, I'm ok with or without reorder-python-imports, so we can remove it if you and @runfalk want. No need to trip up this PR for that. Though I'd prefer we keep it.

Reply:
I would prefer to keep it too, given that I like it (which is why I added it) and how much work it would be to remove it (I'd have to redo half the PR).

My vote is to keep. This PR is ready if we do, though I don't understand why tests are failing on OS X, I think it's unrelated.

Reply:
macOS tests are flaky during to the environment setup (before anything lektor related is done). You can retrigger the tests if you want to be sure.

Though I'm still not sold on the import split I'm not gonna hold up the PR if no one else dislikes it. Maybe it's an aquired taste :laughing:

Reply:
I can certainly say I've acquired it and now prefer the determinism, @runfalk.